examples/multi-document-agentic-rag/README.md

```diff
@@ -0,0 +1,19 @@
+# Multidocument Agentic RAG
+
+![alt text](../../assets/multidocument-agentic-rag.png)
+
+This example provides a comprehensive guide on creating a Multi-Document Agentic RAG leveraging the power of Embeddings and VectorDB. We'll explore how we can use Reasoning + Acting (ReAct) strategy to harness our RAG setup and make it more intelligent using different tools.
+
+Colab walkthrough - <a href="https://colab.research.google.com/github/lancedb/vectordb-recipes/blob/main/examples/Multidocument-Agentic-RAG/main.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> 
```

Comment:
Colab link will break due to wrong folder name

Reply:
Fixed